File: output/CodeGen/codenet/C++/Java/s240627521.java
public class s240627521 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            a[i] = in.nextInt();
        }
        Arrays.sort(a);
        int max = Integer.MIN_VALUE;
        for (int i = 1; i < n; i++) {
            if (max > Math.abs(a[0] - 2 * a[i])) {
                max = Math.abs(a[0] - 2 * a[i]);
                System.out.println(a[0] + " " + a[i]);
            }
        }
    }
}

File: output/CodeGen/codenet/C++/Python/s240627521.py
def main():
  N = int(raw_input())
  A = [int(i) for i in raw_input().split()]
  A.sort(reverse=True)
  max = inf
  for i in xrange(1, N):
    if max > abs(A[0] - 2 * A[i]):
      max = abs(A[0] - 2 * A[i])
      index = i
  print A[0], A[index]

main()